clientconn.go

```diff
@@ -150,6 +150,8 @@ func WithWriteBufferSize(s int) DialOption {
 
```

Comment:
Should we also do the same thing with the write buffers while we're at it?

Reply:
Done.